cafebabel/templates/users/detail.html

```diff
@@ -22,7 +22,12 @@ <h2>About</h2>
   {% endif %}
 
   <ul class=info-profile>
-    <li><span>{{ user.profile.name or '' }}</span></li>
+    <li>
+      <span>{{ user.profile.name or '' }}</span>
+      {% if user.is_me() or current_user.has_role('editor') %}
+        / <span><a href="mailto:{{ user.email }}">{{ user.email }}</a></span>
```

Comment:
Why the prepending `/`?

Reply:
The separator is left at @cafebabel-webmaster discretion ;)